### PR TITLE
docs(aio): added DevExtreme to the UI Components section

### PR DESCRIPTION
**Pull request by donwibier**
## PR Checklist
Does please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
DevExtreme is missing from the resources.json UI Components section

Issue Number: N/A

## What is the new behavior?
DevExtreme has been added to the resources.json UI Components section

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by donwibier** [comment]
I signed it!

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -227,6 +227,12 @@
       "UI Components": {
         "order": 4,
         "resources": {
+          "DevExtreme": {
+            "desc": "50+ UI components including data grid, scheduler, charts, editors, maps and other multi-purpose controls for creating highly responsive web applications for touch devices and traditional desktops.",
+            "rev": true,
+            "title": "DevExtreme ",
+            "url": "https://js.devexpress.com/Overview/Angular/"
```
Comment (gkalpak) [inline aio/content/marketing/resources.json]: This URL redirects to `https://js.devexpress.com/` :confused:

**Comment by gkalpak** [comment]
Also, the commit message does not follow the [commit message guidelines][1].

[1]: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

File: aio/content/marketing/resources.json (hunk 1 above)
Comment (donwibier) [inline aio/content/marketing/resources.json]: Yes, the link will be redirected very soon to the specific Angular section.
I will put some pressure on it at our team

**Comment by donwibier** [comment]
Yes, I'm sorry, I noticed the commit guidelines too late :-o (after committing and pushing)
I hope I did a better job on the pull-request :-)

**Comment by donwibier** [comment]
I have just changed the commit note to follow your guidelines on my fork. Do you want me to do another pull request?

File: aio/content/marketing/resources.json (hunk 1 above)
Comment (dxbykov) [inline aio/content/marketing/resources.json]: The link is updated.

**Comment by gkalpak** [comment]
Can you squash all three commits together and force push to your branch?

Also the commit message type needs to be `docs` (not `doc`) or our linter will bark :smiley:
E.g. something like:

> docs(aio): add DevExtreme to the UI Components section of resources.json

**Comment by donwibier** [comment]
I have updated the commit message to docs(aio) : .... and the link is pointing to our github repo for DevExtreme

**Review by StephenFluin — APPROVED** [review]

**Comment by gkalpak** [comment]
I accidentally caused this to be closed while trying to squash the commits and fix the commit message 🤕
Opened #17939 instead. (Sorry, @donwibier 😇)

**Pull request closed without merging**